binding/jms/src/test/java/io/tracee/binding/jms/TraceeMessageListenerTest.java

```diff
@@ -47,22 +47,31 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
 
 	@Test
 	public void testDecodesFromMessage() throws Exception {
-		encodedContext.put("contextFromMessage","yes");
+		encodedContext.put(INVOCATION_ID_KEY, "yes");
 		unit.intercept(invocationContext);
-		assertThat(backend.getValuesBeforeLastClear(), hasEntry("contextFromMessage", "yes"));
+		assertThat(backend.getValuesBeforeLastClear(), hasEntry(INVOCATION_ID_KEY, "yes"));
+		verify(mdbLike).onMessage(message);
 	}
 
 	@Test
 	public void testCleansUpContextAfterProcessing() throws Exception {
-		encodedContext.put("contextFromMessage","yes");
+		encodedContext.put("contextFromMessage", "yes");
 		unit.intercept(invocationContext);
 		assertThat(backend.isEmpty(), is(true));
+		verify(mdbLike).onMessage(message);
+	}
+
+	@Test
+	public void testInvocationIdGeneratedIfAbsent() throws Exception {
+		unit.intercept(invocationContext);
```

Comment:
Hey @Hippoom,

if you add `encodedContext.put("contextFromMessage","yes");` as first line in your test method all other tests like `#testDecodesFromMessage`, `#testCleansUpContextAfterProcessing` and (that's the reason for my appeal) the assertion in `MdbLike` is still working and shouldn't be deleted.

I agree, that the assertion in `MdbLike` is not scope of your test but removing it is a degration of the other test cases in my opinion.